akka-docs/src/test/scala/docs/io/ScalaUdpMulticastSpec.scala

```diff
@@ -63,7 +68,7 @@ class ScalaUdpMulticastSpec
             // unbind
             system.stop(listener)
           }
-        }
+        } should ===(true)
```

Comment:
wouldn't it be more natural to just re-throw the AssertionError instead of this dance with boolean flag?

Reply:
I agree, that's not perfect either because it will only be the last interface that it tried which of none suceeded, but it's better than this, I was just lazy, I'll fix that.

Reply:
To clarify: there is no way to know which interfaces on each platform that supports multicast, with wifi and docker virtual interfaces and whatnot (naming is different across platforms etc), so what this test does is to enumerate all ipv6 capable interfaces and tries each, at least one of them should succeed or else this test fails.